.github/workflows/main.yml

```diff
@@ -42,6 +44,12 @@ jobs:
           build \
           jacocoRootReport \
           prepareDocsForUploadToGhPages
+    - name: Generate build provenance attestations
+      uses: actions/attest-build-provenance@49df96e17e918a15956db358890b08e61c704919 # v1.2.0
+      with:
+        subject-path: |
+          'junit-jupiter/build/libs/junit-jupiter-*-SNAPSHOT.jar'
+          'junit-jupiter-api/build/libs/junit-jupiter-api-*-SNAPSHOT.jar'
```

Comment:
I think this should be part of the `publish_artifacts` job below instead.

Reply:
Moved accordingly.